addons/rose/addon/components/rose/form/fieldset/index.hbs

```diff
@@ -0,0 +1,21 @@
+<fieldset ...attributes class='rose-form-fieldset' aria-describedby={{concat 'title-' this.id}}>
+    <legend 
+        id={{concat 'title-' this.id}}
+        class='rose-form-fieldset-title'
+    >
+        {{yield to='title'}}
+    </legend>
+    
+    {{#if (has-block "description")}}
```

Comment:
@randallmorey this fix the issue of rendering an empty `<div>` is description is not provided.